.github/workflows/run-test-build.yml

```diff
@@ -0,0 +1,15 @@
+# Runs on any new commit or commits pushed to any branch. Intended for testing new 
```

Comment:
this is every new push, right?  Not every new commit?  Ie, if I push 5 commits at once, I assume that'd trigger just one build, on the last commit.